src/FINDOLOGIC/Api/Responses/Xml21/Xml21Response.php

```diff
@@ -2,17 +2,121 @@
 
 namespace FINDOLOGIC\Api\Responses\Xml21;
 
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
 use FINDOLOGIC\Api\Responses\Response;
+use FINDOLOGIC\Api\Responses\Xml21\Properties\Servers;
+use FINDOLOGIC\Api\Responses\Xml21\Properties\LandingPage;
+use FINDOLOGIC\Api\Responses\Xml21\Properties\Promotion;
+use FINDOLOGIC\Api\Responses\Xml21\Properties\Results;
+use FINDOLOGIC\Api\Responses\Xml21\Properties\Product;
+use SimpleXMLElement;
 
 class Xml21Response extends Response
 {
+    /** @var Servers $servers */
+    private $servers;
+
+    /** @var LandingPage|null $landingPage */
+    private $landingPage;
+
+    /** @var Promotion|null $promotion */
+    private $promotion;
+
+    /** @var Results $results */
+    private $results;
+
+    /** @var Product[] $products */
+    private $products = [];
+
     public function __construct($response, $responseTime = null)
     {
         parent::__construct($response, $responseTime);
     }
 
     protected function buildResponseElementInstances($response)
```

Comment:
Add a typehint in what `$response` can/should be.

Reply:
Is this covered by change to use @inheritDoc?

Reply:
Typehint this method with `@interhitDoc` and in `\FINDOLOGIC\Api\Responses\Response` make sure that `@param $response` is typehinted as `string`.